### PR TITLE
Cleanup selecting windows.

### PR DESCRIPTION
**Pull request by pekkaklarck**
Code and documentation cleanup to Select Window. Also add Get Window Handles to replace List Windows. See individual commit messages for more details. Ready for review.

### CODE REVIEW

File: src/SeleniumLibrary/keywords/window.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+        when doing something on an existing window. It is not possible to
+        select windows opened with `Open Browser`, `Switch Browser` must
+        be used instead. Notice also that alerts should be handled with
+        `Handle Alert` or other alert related keywords.
+
+        The ``locator`` can be specified using different strategies somewhat
+        similarly as when `locating elements` on pages.
+
+        - By default the ``locator`` is matched against window handle, name,
+          title, and URL. Matching is done in that order and the the first
```
Comment (aaltat) [inline src/SeleniumLibrary/keywords/window.py:48]: Comma after word title is not needed.

File: src/SeleniumLibrary/keywords/window.py [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        - If the ``locator`` is ``CURRENT`` (case-insensitive), nothing is
+          done. This effectively just returns the current window handle.
+
+        - If the ``locator`` is not a string, it is expected to be a list
+          of window handles _to exclude_. Such a list of excluded windows
+          can be get from `Get Window Handles` prior to doing an action that
+          opens a new window.
+
+        Example:
+        | `Click Link`      | popup1      |      | # Open new window |
```
Comment (aaltat) [inline src/SeleniumLibrary/keywords/window.py:73]: Nitpick, should the tables be same size?

File: src/SeleniumLibrary/keywords/window.py (hunk 1 above)
Comment (pekkaklarck) [reply src/SeleniumLibrary/keywords/window.py:48]: Not needed but recommended: http://www.getitwriteonline.com/archive/052709serialcomma.htm

File: src/SeleniumLibrary/keywords/window.py (hunk 2 above)
Comment (pekkaklarck) [reply src/SeleniumLibrary/keywords/window.py:73]: Can you clarify?

**Pull request merged**